aiidalab_widgets_base/computational_resources.py

```diff
@@ -225,33 +183,18 @@ def _setup_new_code(self, _=None):
             clear_output()
             if self.btn_setup_new_code.value:
                 self._setup_new_code_output.layout = {
-                    "width": "500px",
+                    "width": "460px",
```

Comment:
Let's remove this here and inject it into the widget definition.

Reply:
I guess you mean only the `width` not the whole layout as a dictionary? See my changes, and let me know if it is correct.

Reply:
yep, you did what I had in mind 👍